public/data/SulthanNK.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Sulthan Mohaideen",
+  "bio": "Tech Savvy | Engineer",
+  "avatar": "https://github.com/SulthanNK.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/SulthanNK",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/SulthanNK",
+      "icon": "twitter"
+    }
+  ]
+}
```

Comment:
```suggestion
}

```

Reply:
This suggestion should be correct now, then we can merge

Reply:
hey eddie we need to add extra space ?